index.html

```diff
@@ -62,37 +62,47 @@ <h3 class="meal-name">
                     </div>
                 {{~}}
             </div>
-            
         </script>
     </head>
-    
+
     <body>
         <div class="top-container">
-            <h1>TTY:n ruokalistat</h1>
-            
+            <div>
+                <div id="tunilogo"></div>
+
+                <h1>Ruokalistat</h1>
+            </div>
+
             <button id="prev-day" class="nav-button">Edellinen päivä</button>
             <button id="next-day" class="nav-button">Seuraava päivä</button>
         </div>
-        
-        <div id="main-content">
-            
-        </div>
-                
+
+        <div id="main-content"></div>
+
         <!-- build:js js/build.js -->
         <script src="lib/dot.min.js"></script>
         <script src="lib/moment.min.js" charset="utf-8"></script>
         <script src="lib/moment.fi.js" charset="utf-8"></script>
         <script src="js/main.js" charset="utf-8"></script>
         <!-- endbuild -->
-        
+
         <script>
-            (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-            (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-            })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+            ;(function(i, s, o, g, r, a, m) {
```

Comment:
Why?

Reply:
Auto formatting by linter and prettifier. If this is not wanted maybe the build process should minify it or adding linting and formatting rules to project.